cleanlab/object_detection/rank.py

```diff
@@ -169,11 +175,12 @@ def issues_from_scores(label_quality_scores: np.ndarray, *, threshold: float = 0
 def _compute_label_quality_scores(
     labels: List[Dict[str, Any]],
     predictions: List[np.ndarray],
-    aggregation_weights: Optional[Dict[str, float]] = None,
-    *,
     method: str = "objectlab",
```

Comment:
method param should go below `*`

Reply:
https://github.com/cleanlab/cleanlab/blob/420e69bed2a1e64c6bf282b7da54359465361694/cleanlab/object_detection/rank.py#L175